fake_spectra/spectra.py

```diff
@@ -760,20 +773,26 @@ def compute_spectra(self, elem, ion, ll, get_tau):
         sigma_X is the cross-section for this transition.
         """
         #Get array sizes
-        nsegments = self.snapshot_set.get_n_segments()
+        nsegments = self.snapshot_set.get_n_segments(part_type=0)
         arepo = (self.kernel_int == 2)
+        if arepo :
+           nsegments=1
```

Comment:
I understand this as meaning that with the Voronoi mesh reconstruction there is no MPI splitting? Because the meshes mean that particles are not independent. Could you document this somewhere in Readme if so?